server/src/main/java/com/flipkart/varadhi/web/v1/TopicHandlers.java

```diff
@@ -71,9 +75,10 @@ public void get(RoutingContext ctx) {
     public void create(RoutingContext ctx) {
         //TODO:: Enable authn/authz for this flow.
         //TODO:: Consider using Vertx ValidationHandlers to validate the request body.
-        //TODO:: Consider reverting on failure and transaction kind of semantics for all operations.
+        //TODO:: Consider reverting on failure and ≠≠ kind of semantics for all operations.
 
         TopicResource topicResource = ctx.body().asPojo(TopicResource.class);
+        topicResource.validate();
```

Comment:
see if we can perform validation as part of .asPojo() extension method as indicated earlier comment.

Reply:
done